devops/core/views.py

```diff
@@ -20,6 +20,7 @@ def sync(request):
 
 
 def home(request):
+    return HttpResponseRedirect('/static/dash.html')
```

Comment:
Right now, all the functionality is in the rich static app. That is not expected to be true forever. But until it is false, we are treating the static app as the home. so we just redirect home to it. It isn't elegant, but it works for now.